Lombiq.ChartJs.Tests.UI/Lombiq.ChartJs.Tests.UI.csproj

```
@@ -0,0 +1,46 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Title>Lombiq Chart.js for Orchard Core - UI Test Extensions</Title>
+    <Authors>Lombiq Technologies</Authors>
+    <Copyright>Copyright © 2022, Lombiq Technologies Ltd.</Copyright>
+    <Description>Lombiq Chart.js for Orchard Core - UI Test Extensions: Extensions to aid in UI testing Lombiq Chart.js for Orchard Core.</Description>
+    <PackageIcon>NuGetIcon.png</PackageIcon>
+    <PackageTags>OrchardCore;Lombiq;AspNetCore;Chart.js</PackageTags>
+    <RepositoryUrl>https://github.com/Lombiq/Orchard-Chart.js</RepositoryUrl>
+    <PackageProjectUrl>https://github.com/Lombiq/Orchard-Chart.js/Lombiq.ChartJs.Tests.UI/Readme.md</PackageProjectUrl>
+    <PackageLicenseFile>License.md</PackageLicenseFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="Assets\bar_chart.dib" />
+    <None Remove="Assets\line_chart.dib" />
+  </ItemGroup>
```

```suggestion
  <ItemGroup>
    <None Remove="Assets\*.dib" />
  </ItemGroup>
```

Same [below](https://github.com/Lombiq/Orchard-Chart.js/pull/21/files#diff-270989268358348e9dde7d4120c99e6afd2b7d0b19d733db7fdadebf32b89b6cR35-R36). Also these should be next to each other.